### ISSUE TITLE
"Accessibility: true" causes scroll jump to slick on click and viewport changes

### ISSUE THREAD

**Issue by karedas** [kenwheeler/slick#3020]
Good morning and sorry for my english.

After the latest updates (now I have 1.7.1), clicking on slides or change the viewport using the "breakpoint" property, let the browser jump with a scroll at the slick position.
Is the same issue posted here some V. ago and marked resolved https://github.com/kenwheeler/slick/issues/1537.

The only fix  found is to enter the parameter "accessibility: false".

On I.E Edge, Chrome, Firefox.

Thanks!

**Comment by leggomuhgreggo** [comment]
Hey @karedas thank you for reporting the issue.

You may refer to this discussion [here](https://github.com/kenwheeler/slick/pull/2119#issuecomment-308934193).

Do you have an example where you're seeing the issue?

A11Y rules indicate that focus should be set to a new slide. 

We disabled the focus of a new slide if `autoplay: true`, but thought that if a user is deliberately advancing a slide, the focus (and scroll jump) was acceptable, since presumably they'd be interacting with the slider that was being focused.

I appreciate your feedback

**Comment by karedas** [comment]
Hi and thanks for the answer.
I wanted to show you an example via codepen but did not find a link to cdn for 1.7.1.
However, the biggest problem occurs when changing a window viewport, as slick correctly recalculates the number of slides present (as I have indicated), but it is at that precise moment that it force the jump on him.  producing on long pages the unwanted effect.

For example, I have a long page and when I resize my browser I jump to its position.
As I said, the only way I found to ignore the error is to set "accessibility: false" as indicated in the previous report.

This is my code (Fzlib is a globe scope from my webpack config library).

```
	$(_.activeTab).slick({
	infinite: false,
	arrows: false,
	slidesToShow: 6,
	slidesToScroll: 6,
	dots: false,
	speed: 300,
	responsive: [{
			breakpoint: parseInt(Fzlib.app.config.breakpoints.screenMdMax),
			settings: {
				slidesToShow: 3,
				slidesToScroll: 3,
				infinite: true,
				dots: true
			}
		},
		{
			breakpoint: parseInt(Fzlib.app.config.breakpoints.screenXsMax),
			settings: {
				slidesToShow: 2,
				slidesToScroll: 2,
				infinite: false,
				dots: true
			}
		},
	]
});
```

When I reduce the size of the browser window (the viewport), slick forces a jump on him (I think for a focus event).

Thanks again.

**Comment by karedas** [comment]
Sorry I read the reference to the other discussion.
Thank you and sorry yet.

**Comment by karedas** [comment]
In your slick code i found this in the postSlide prototype:

```
....
            if (_.options.accessibility === true) {
                _.initADA();
                // for non-autoplay: once active slide (group) has updated, set focus on first newly showing slide
                if (!_.options.autoplay) {
                    var $currentSlide = $(_.$slides.get(_.currentSlide));
                    $currentSlide.attr('tabindex', 0).focus();
                }
            }
...
```
Commenting on the line where trigger the focus event (1692) i can fix my problem. 
I do not know if it can help to understand my problem :( .

**Comment by leggomuhgreggo** [comment]
@karedas I updated the CDN link last night, so that should work now if you're still interested. Sorry about the delay in getting it working.

The code you posted is definitely what's causing the jump. I think it might be worth adding an option for `focusOnChange`. It will break a11y compliance, but I can see the jump being enough of a usability issue that it's necessary.

**Comment by pdscti** [comment]
I'm seeing this jump when using slick 1.7.1's min file on a slideshow that is using autoplay. If you visit http://www.hilldickinson.com/gdpr-solution and move the browser window to halfway down the slideshow, you'll get a jump back up to the top of the slide window when the slide changes. This occurs without any user interaction.

**Comment by leggomuhgreggo** [comment]
Thanks @pdscti I'll look into this today

**Comment by leggomuhgreggo** [comment]
@pdscti Hmm, are you triggering slide change manually? Autoplay shouldn't trigger the jump. [Here's a codepen](https://codepen.io/leggomuhgreggo/pen/qXaBoO).

**Comment by pdscti** [comment]
@leggomuhgreggo Sorry about that: it looks like we are: somebody else's code and they've manually implemented a timer and .slick(slickNext) in the javascript from the look of it.

**Comment by rwam** [comment]
Have this jump after updating to 1.7.1 too and I have also weird outline styles based on my user agent stylesheet (chrome). Ok, the styles I can fix with custom styles but I think this should maybe part of the slick.css, shouldn't it?

I would subscribe an option like mentioned by @leggomuhgreggo but for the moment I comment out the relevant lines to avoid setting the focus.
